mrcnn/model.py

```diff
@@ -1238,6 +1238,33 @@ def mrcnn_mask_loss_graph(target_masks, target_class_ids, pred_masks):
     return loss
 
 
+def image_class_loss_graph(target_image_class, pred_image_class):
+    """Loss for the image classifier head of Mask RCNN.
+
+    target_image_class: [batch, num_image_classes]
+    pred_image_class: [batch, num_image_classes]. Uses zero padding to fill in the array.
+    """
+
+    non_zeros = tf.cast(tf.reduce_max(target_image_class, axis=1) > 0, 'float32')
+
+    # Loss
+    loss = tf.nn.softmax_cross_entropy_with_logits(
```

Comment:
We don't have logits here but probs. TF doc even says:
> Warning: This op expects unscaled logits, since it performs a softmax on logits internally for efficiency. Do not call this op with the output of softmax, as it will produce incorrect results.

I suggest `K.categorical_crossentropy(target_image_class, pred_image_class)`